### PR TITLE
DPC-740: Retrieve BFD IDs and MBIs in 1-800-MEDICARE ETL process

### PR DESCRIPTION
**Pull request by embh**
Draft. This likely needs to be on hold until IDs are sorted out.

**Why**

Explain the motivation for this PR.

**What Changed**

Describe the changes made to the code base including new features, refactoring of existing services and user visible API changes.

**Choices Made**

Justify changes made and any reasons for why a given path was taken, as opposed to an alternative option.

**Tickets closed**:

Give a list of tickets closed in this PR.

**Future Work**

List any additional tickets that have either been created due to work in this PR, or existing tickets that expand upon the feature or provide additional fixes.

**Checklist**

- [ ] All tests are passing via `make ci-app` (app change) and `make ci-web` (website change)
- [ ] Swagger documentation has been updated
- [ ] FHIR documentation has been updated
- [ ] Any required dpc-ops changes have a PR submitted and mentioned in this ticket
- [ ] Any manual migration steps are documented, scripts written (where applicable), and tested
- [ ] Before merging, any required dpc-ops changes have been approved and merged into master of the dpc-ops repo

### CODE REVIEW

File: dpc-bluebutton/src/main/java/gov/cms/dpc/bluebutton/client/BlueButtonClientImpl.java [hunk 1]
```diff
@@ -181,4 +213,15 @@ public CapabilityStatement requestCapabilityStatement() throws ResourceNotFoundE
             timerContext.stop();
         }
     }
+
+    protected static String hashHICN(String hicn, String pepper, int iterations) {
+        KeySpec keySpec = new PBEKeySpec(hicn.toCharArray(), Hex.decode(pepper), iterations, 256);
```
Comment (dkrylovsb) [inline dpc-bluebutton/src/main/java/gov/cms/dpc/bluebutton/client/BlueButtonClientImpl.java]: I _think_ key length should be `32`.

Comment (embh) [reply dpc-bluebutton/src/main/java/gov/cms/dpc/bluebutton/client/BlueButtonClientImpl.java]: It is 256 bits/32 bytes.

Comment (dkrylovsb) [reply dpc-bluebutton/src/main/java/gov/cms/dpc/bluebutton/client/BlueButtonClientImpl.java]: Thanks!

**Comment by embh** [comment]
This PR is stale. Will create a new one for this process after MBI handling is complete.

**Pull request closed without merging**